tests/store/test_footprints.py

```diff
@@ -1,10 +1,32 @@
 import pytest
+from functools import partial
 from helpers import get_footprint_datapath
 from openghg.retrieve import search
 from openghg.objectstore import get_bucket
-from openghg.store import Footprints, load_metastore
+from openghg.store import Footprints
 from openghg.util import hash_bytes
-from openghg.standardise import standardise_footprint
+from openghg.standardise._standardise import standardise_footprint, standardise_from_binary_data
+
+@pytest.fixture(autouse=True, scope="module")
+def auto_bucket():
+    global bucket
+    bucket = get_bucket()
+    yield bucket
+    globals().pop('bucket')
+
+
+@pytest.fixture(autouse=True)
```

Comment:
In this case, is the autouse producing anything here (based on your previous comments around the reason for global bucket)?

It looks like `standardise` is used within the tests as needed as an input?

Reply:
I think you're right, I removed the "auto_bucket" fixture and standardise fixture and just called bucket where needed.